### ISSUE TITLE
Minmax Standardization

### ISSUE THREAD

**Issue by cnacc23** [anbai106/mlni#11]
Hi Hao, 

 I just had a couple follow-up questions after accessing the joblib files: 

- I noticed that the training data is standardized using the minmax method in the regression_roi function. I had previously normalized my entire data set (both the training and test sets) using scipy. Seeing as the training data is standardized within the function, would you say that it is redundant/not worth preforming the initial normalization that I did? Secondly, I am just confirming that I should apply the same minmax standardization to the test set to maintain consistency with the training set before running the model.

- The output of regression_roi contains two models: the single and mean version. Why are there two models? Should I choose which one to use based on the best MAE and r score or should I stick to just using the mean model (assuming that it is the mean across all iterations).

Thank you so much, 
Cienna

**Comment by anbai106** [comment]
Hi, 

Answers to your questions:
1: You should use the same standardization method when you train the model. That is, you need to use the source code in the package for your external data. Otherwise, the model cannot generalize to independent data. I forgot the detail, but by default it is minmax standardization.

2: You can use either method, which just saves the final models in different ways.
